app/assets/stylesheets/donor.css.scss

```diff
@@ -0,0 +1,182 @@
+.donation-cancel {
+  color: $red;
+}
+
+.donation-container {
+  min-height: 0px;
+}
+
+.donation-container, .profile-container {
+  background: $white;
+  font-size: 12px;
+  margin: 0 auto;
+  padding: 15px;
+  width: 68%;
+}
+
+.donation-container b, .profile-container b {
+  font-weight: 600;
+}
+
+.profile-container {
+  min-height: 350px;
+}
+
+.donation-container h1, .profile-container h1 {
+  background: $medium-green;
+  color: $white;
+  display: block;
+  font-size: 18px;
+  font-weight: normal;
+  margin-bottom: 20px;
+  padding: 10px;
+  text-align: center;
+}
+
+.donation-item {
+  background: $donor-profile-light-grey;
+  margin: 10px 0px;
+  padding: 10px;
+  width: 100%;
+}
+
+.donation-item h1 {
+  background: none;
+  color: $medium-green;
+  font-size: 15px;
+  margin: 0px 0px 10px 0px;
```

Comment:
Shorthand form for property `margin` should be written more concisely as `0px 0px 10px` instead of `0px 0px 10px 0px`<br>`0px` should be written without units as `0`